samples/katib/early-stopping.ipynb

```diff
@@ -551,7 +551,8 @@
     }
    ],
    "source": [
-    "kfp.Client().create_run_from_pipeline_func(median_stop, arguments={})"
+    "from kfp_tekton._client import TektonClient",
```

Comment:
you need a newline for this code, otherwise jupyter notebook will not able to run this line

Reply:
also at the top, run `!pip install kfp-tekton==0.4.0` to let it run on Kubeflow 1.2 with tekton.